doc/man/8/ceph.rst

```diff
@@ -476,7 +482,7 @@ will take its place.
 
 Usage::
 
-  ceph mgr fail <name>
+  ceph mgr fail <role>
```

Comment:
Accidental find/replace. We're not modifying `ceph mgr` in this PR.

Reply:
It was not accidental. I have made the change purposefully. I have noticed this in mgr and I wanted the reviewer to verify it and approve / disapprove.